ocaml/tests/dune

```diff
@@ -2,6 +2,16 @@
   (name suite_alcotest)
   (package xapi)
   (flags (:standard -warn-error +a-3-4-6-9-27-28-29-52))
+  (modules (:standard \
+   test_daemon_manager 
```

Comment:
I think this is getting to the point where we should consider moving test executables to their own subfolder (so that we can avoid having to adding exceptions to this list)

Reply:
Okay, thank you. I'll try to move them.

Reply:
I think having subdirectoriesper area being tested would make sense, like all the clustering tests being in `ocaml/test/cluster` and `tests/network` containing `test_vlan` `test_network_sriov` and storage